e2e/testdata/fn-render/out-of-place-dir/.expected/exec.sh

```diff
@@ -16,4 +16,6 @@
 
 set -eo pipefail
 
+rm -rf out
+
 kpt fn render -o out
```

Comment:
I think we need three test cases:

- fn render fails with `-o existing-dir`
- fn eval fails with `-o existing-dir`
- fn sink fails with `-o existing dir`
-

Reply:
Added.